src/tracker/index.js

```diff
@@ -40,6 +40,12 @@ export default class Tracker {
       repo,
       accept: 'application/vnd.github.v3+json',
     };
+
+    this.createLabel({
```

Comment:
Move this call inside an `initialize` function which should be executed by the caller.

```
async initialize() {
    await this.createLabel({
      name: config.get('tracker.githubIssues.label'),
      color: config.get('tracker.githubIssues.color'),
      description: config.get('tracker.githubIssues.description'),
    });
}
```